src/Umbraco.Cms.Imaging.ImageSharp/Media/ImageSharpImageUrlGenerator.cs

```diff
@@ -47,57 +65,63 @@ public ImageSharpImageUrlGenerator(Configuration configuration)
+            queryString.Add(CropWebProcessor.Coordinates, FormattableString.Invariant($"{crop.Left},{crop.Top},{crop.Right},{crop.Bottom}"));
         }
 
-        if (options.FocalPoint is not null)
+        if (options.FocalPoint is FocalPointPosition focalPoint)
         {
-            queryString.Add(ResizeWebProcessor.Xy, FormattableString.Invariant($"{options.FocalPoint.Left},{options.FocalPoint.Top}"));
+            queryString.Add(ResizeWebProcessor.Xy, FormattableString.Invariant($"{focalPoint.Left},{focalPoint.Top}"));
         }
 
-        if (options.ImageCropMode is not null)
+        if (options.ImageCropMode is ImageCropMode imageCropMode)
         {
-            queryString.Add(ResizeWebProcessor.Mode, options.ImageCropMode.ToString()?.ToLowerInvariant());
+            queryString.Add(ResizeWebProcessor.Mode, imageCropMode.ToString().ToLowerInvariant());
         }
 
-        if (options.ImageCropAnchor is not null)
+        if (options.ImageCropAnchor is ImageCropAnchor imageCropAnchor)
         {
-            queryString.Add(ResizeWebProcessor.Anchor, options.ImageCropAnchor.ToString()?.ToLowerInvariant());
+            queryString.Add(ResizeWebProcessor.Anchor, imageCropAnchor.ToString().ToLowerInvariant());
         }
 
-        if (options.Width is not null)
+        if (options.Width is int width)
         {
-            queryString.Add(ResizeWebProcessor.Width, options.Width?.ToString(CultureInfo.InvariantCulture));
+            queryString.Add(ResizeWebProcessor.Width, width.ToString(CultureInfo.InvariantCulture));
         }
 
-        if (options.Height is not null)
+        if (options.Height is int height)
         {
-            queryString.Add(ResizeWebProcessor.Height, options.Height?.ToString(CultureInfo.InvariantCulture));
+            queryString.Add(ResizeWebProcessor.Height, height.ToString(CultureInfo.InvariantCulture));
         }
 
         if (furtherOptions.Remove(FormatWebProcessor.Format, out StringValues format))
         {
-            queryString.Add(FormatWebProcessor.Format, format[0]);
+            queryString.Add(FormatWebProcessor.Format, format.ToString());
         }
 
-        if (options.Quality is not null)
+        if (options.Quality is int quality)
         {
-            queryString.Add(QualityWebProcessor.Quality, options.Quality?.ToString(CultureInfo.InvariantCulture));
+            queryString.Add(QualityWebProcessor.Quality, quality.ToString(CultureInfo.InvariantCulture));
         }
 
         foreach (KeyValuePair<string, StringValues> kvp in furtherOptions)
         {
             queryString.Add(kvp.Key, kvp.Value);
         }
 
-        if (options.CacheBusterValue is not null && !string.IsNullOrWhiteSpace(options.CacheBusterValue))
+        if (options.CacheBusterValue is string cacheBusterValue && !string.IsNullOrEmpty(cacheBusterValue))
+        {
+            queryString.Add("v", cacheBusterValue);
```

Comment:
The cache buster value is now using the `v` parameter, since it represents the version (or 'just' a value, not a random value like `rnd` suggested) and makes the URL slightly shorter.